.gitignore

```diff
@@ -10,6 +10,7 @@
 fuzzer-code/*.json
 fuzzer-code/*.taint
 pin-*
+pin*
```

Comment:
Hi @PBearson thanks for updating the repo. Really appreciate. Its been a while since we wrote .gitignore. I am wondering why pin* is needed, when we have pin-* ? All other changes are good. I will merge them once we know about this pin* addition. thank you :)

Reply:
The reason I added pin* is because the 32-bit version of VUzzer specifies that "the [pin] framework must be present in directory pin inside the VUzzer top directory." I just assumed that renaming the $PIN_HOME directory to pin/ may be recommended.

Reply:
aha.. yes, we do set PIN_HOME and PIN_ROOT now.

Reply:
Since pin* includes pin-*, I just removed pin-*. Let me know if this is okay.